<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>returnItem</class>
 <widget class="QDialog" name="returnItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-radius: 5px;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>14</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 5px;</string>
   </property>
   <property name="title">
    <string>RETURN LIBRARY ITEM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Roll Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Lib Item ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Return Date</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Return item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color:rgb(255, 2, 61);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Checking if there is any fine to pay </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color: rgb(170, 170, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Pay fine</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>You have to pay a fine of</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Fine you are going to pay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color: rgbrgbrgb(255, 255, 255);
background-color:rgb(255, 2, 61);
border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Go back to home</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Rockwell&quot;;
color:rgbrgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
